src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -545,17 +545,26 @@ void userWithTeamsExport(OutputStream output) throws IOException{
         log.info("Exporting USER_TEAMS_DATA to CSV", value(EVENT, CSV_EXPORT_START));
         OutputStream buffer = new BufferedOutputStream(output);
         OutputStreamWriter outputWriter = new OutputStreamWriter(buffer, "UTF-8");
-        List<String> headers = List.of("userUUID", "teamsUUIDs");
+        List<String> headers = List.of("Username", "First Name", "Last Name", "Team Name", "Unit Name");
 
         Set<UserWithTeamsDto> users = infoClient.getUsersWithTeams();
 
         try (CSVPrinter printer = new CSVPrinter(outputWriter, CSVFormat.DEFAULT.withHeader(headers.toArray(new String[0])))) {
             users.forEach((user) -> {
                 try {
-                    List<UUID> userTeams = user.getTeamUUIDs();
-                    String teamUUIDs = userTeams.stream().map(UUID::toString).collect(Collectors.joining(", "));
-                    printer.printRecord(user.getId(), teamUUIDs);
-                    outputWriter.flush();
+                    List<String> teamNames = user.getTeamNames();
```

Comment:
This would require the same change to work with a Map but should look a little cleaner.